gate-core/src/main/groovy/com/netflix/spinnaker/gate/config/AuthConfig.groovy

```diff
@@ -80,6 +80,7 @@ class AuthConfig {
         .antMatchers(PermissionRevokingLogoutSuccessHandler.LOGGED_OUT_URL).permitAll()
         .antMatchers('/auth/user').permitAll()
         .antMatchers(HttpMethod.POST, '/webhooks/**').permitAll()
+        .antMatchers(HttpMethod.POST, '/notifications/callbacks/**').permitAll()
```

Comment:
This needs to be unauthenticated because it will be coming from external services (like Slack), very similar to webhooks.

Reply:
@cfieber, did I convince you of the the need for this the last time we chatted?